src/AssetManagement/utils/ActionAssetData.sol

```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >=0.8.0 <0.9.0;
+
+struct actionAssetsData {
+    address[] assets; // Assets addresses
+    uint256[] assetIds; // Arcadia Protocol asset ids (internal ids given to arcadia's whitelisted assets)
```

Comment:
* @param assetIds The asset IDs that will be withdrawn for ERC721 & ERC1155.
     * When withdrawing an ERC20, this will be disregarded, HOWEVER a value (eg. 0) must be filled!

Reply:
This is simply to make that struct generic for future assets